src/content/packages-and-plugins/swift-package-manager/for-app-developers.md

```diff
@@ -0,0 +1,144 @@
+[Swift packages]: https://swiftpackageindex.com/
+[open an issue]: {{site.github}}/flutter/flutter/issues/new?template=2_bug.yml
+
+{% include docs/swift-package-manager/how-to-enable-disable.md %}
+
+## How to add Flutter to an existing app (add-to-app)
+
+Flutter's Swift Package Manager support doesn't work with add-to-app scenarios
+yet.
+
+For status updates, see [flutter#146957].
+
+[flutter#146957]: https://github.com/flutter/flutter/issues/146957
+
+## How to add Swift Package Manager integration to a project manually
+
+{% tabs %}
+{% tab "iOS project" %}
+
+{% include docs/swift-package-manager/migrate-ios-project-manually.md %}
+
+{% endtab %}
+{% tab "macOS project" %}
+
+{% include docs/swift-package-manager/migrate-macos-project-manually.md %}
+
+{% endtab %}
+{% endtabs %}
+
+## How to use a Swift Package Manager Flutter plugin that requires a higher OS version
+
+If a Swift Package Flutter Manager plugin requires a higher OS version than
+the project, you might get an error like this:
+
+```plaintext
+Target Integrity (Xcode): The package product 'plugin_name_ios' requires minimum platform version 14.0 for the iOS platform, but this target supports 12.0
+```
+
+To use the plugin, you'll need to increase the **Minimum Deployments** of your
+app's target.
+
+:::warning
+This raises your app's minimum OS version requirement.
+:::
+
+{% render docs/captioned-image.liquid,
+image:"development/packages-and-plugins/swift-package-manager/minimum-deployments.png",
+caption:"The target's **Minimum Deployments** setting" %}
+
+## How to add Swift Package Manager integration to a custom Xcode target
+
+Your Flutter Xcode project can have custom [Xcode targets][] to build additional
+products, like frameworks or unit tests.
+You can add Swift Package Manager integration to these custom Xcode targets
+as well.
+
+Follow the steps in
+[How to add Swift Package Manager integration to a project manually][manualIntegration].
+
+In [Step 1][manualIntegrationStep1], list item 6 use your custom target instead
+of the `Flutter` target.
+
+In [Step 2][manualIntegrationStep2], list item 6 use your custom target instead
+of the `Flutter` target.
+
+[Xcode targets]: https://developer.apple.com/documentation/xcode/configuring-a-new-target-in-your-project
+[manualIntegration]: /packages-and-plugins/swift-package-manager/for-app-developers/#how-to-add-swift-package-manager-integration-to-a-project-manually
+[manualIntegrationStep1]: /packages-and-plugins/swift-package-manager/for-app-developers/#step-1-add-fluttergeneratedpluginswiftpackage-package-dependency
+[manualIntegrationStep2]: /packages-and-plugins/swift-package-manager/for-app-developers/#step-2-add-run-prepare-flutter-framework-script-pre-action
+
+## How to remove Swift Package Manager integration
+
+The Flutter CLI migrates your project to add Swift Package Manager integration.
+This makes your project download the Swift packages that your Flutter plugins
+depend on.
```

Comment:
**issue:** 
Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure).

```suggestion
To add Swift Package Manager integration, the Flutter CLI migrates your project.
During this migration, your project downloads the Swift packages
on which your Flutter plugins depend.
```

Reply:
Mm my second sentence was incorrect. Sorry about that.

I updated this to:

> To add Swift Package Manager integration, the Flutter CLI migrates your project.
> This migration updates your Xcode project to add Flutter plugin dependencies.

Let me know what you think!